app/assets/javascripts/application.js

```diff
@@ -43,7 +43,5 @@
 //= require ./src/Metamaps.Mapper
 //= require ./src/Metamaps.Admin
 //= require ./src/Metamaps.Import
+//= require ./src/Metamaps.Header
 //= require ./src/Metamaps.JIT
-//= require_directory ./shims
```

Comment:
@devvmh so many of these things were JUST in here for famous and now we can toss them!